Functions/Shortcuts/Wine/script.js

```diff
@@ -34,7 +34,7 @@ var WineShortcut = function () {
 
     that.miniature = function(miniature) {
       if(isArray(miniature)) {
-          var application = that._appsManager.getApplication(miniature);
+          var application = that._appsManager.cachedRepository().getApplication(miniature);
```

Comment:
I don't like the idea of exposing the fact that you are delegating to a cached repository. This could change later and should not be part of the public API